exporter/awsxrayexporter/internal/translator/segment_test.go

```diff
@@ -53,7 +53,7 @@ func TestClientSpanWithRpcAwsSdkClientAttributes(t *testing.T) {
 	attributes[conventions.AttributeHTTPHost] = "dynamodb.us-east-1.amazonaws.com"
 	attributes[conventions.AttributeHTTPTarget] = "/"
 	attributes[conventions.AttributeRPCService] = "DynamoDB"
-	attributes[conventions.AttributeRPCMethod] = "GetItem"
+	attributes[conventions.AttributeRPCMethod] = "GetDataPoint"
```

Comment:
@bogdandrutu too ambitious of a replace haha

Reply:
:))))